### ISSUE TITLE
Feature Request: Creating bundled js files

### ISSUE THREAD

**Issue by CarstenGrohmann** [QQuick/Transcrypt#760]
The current Javascript bundler cannot merge existing source map files. This means that you can no longer use the source map generated by Transcrypt after the module files have been merged with a bundler.

Please add an option to create an all-in-one js file as well as the sourcemap file like Transcrypt 3.6 did.

**Comment by JennaSys** [comment]
I use Parcel V2 with the [parcel-transformer-transcrypt](https://www.npmjs.com/package/parcel-transformer-transcrypt) plugin and the source maps that Transcrypt generates get bundled successfully. I also did a full [write-up](https://dev.to/jennasys/using-parcel-v2-and-transcrypt-to-create-web-applications-with-python-30ac) on using the Transcrypt plugin for Parcel V2 as well. If that approach works for your purpose, I'll close this issue out.

**Comment by JennaSys** [comment]
Good idea. The [section in the docs about using Parcel](https://www.transcrypt.org/docs/html/integration_javascript.html?highlight=parcel#example-using-the-parcel-js-bundler-to-package-a-set-of-modules-written-in-diverse-programming-languages) could use a bit more clarification in general.
